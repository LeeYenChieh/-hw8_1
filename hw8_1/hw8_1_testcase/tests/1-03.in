RfDvilwMTgA4fhK56S482I loBCUWn8xoadU7Q1R07ZfcYqdLmlHIFAUsXgV4iT4b3Xv8RZyytCfhuqmIRPFM8
WVsF6HcR EtaBkD9thdKbTA7NJRzVvxhhpAHZMjP7T3dPjYl9cCPawIYYjfPnSl4
znZNPypqsOgq2hHCs3QrALRrkucKSLGiKdKI4ANruh4wlqlHfRz VcjK43jwmLR16gRgLxIVy0dCx1DXTfSZdtJDHcvXi3MHt7c
cAvqp5kbXGPMsIswkJeA4MRvyRKKj9pkv5HQ14Xajv6DqFuvX49HoGrdZ6r6I 1ZOj2txr8JAGRMtE4nhl18cVWsSWO2BzL6UAdOxEXBfCj6yx5jsgsF6kPr
TU7acnNYAQ6lZsVK0tXp2gGb7TKVBEAeDsQdSBWrhCFFclQpCWpHnB v3ufhmESNlGwdhq1wP9uCBOrym6lU8tQtbIPuJnLVkVrJ97gaw2UxtcXx5
5hTt3mh pnRjKn
oXWPHPfa878dWRDekZf5kxAaRuhUmSB6C5RBrmUSefBdu9 WLT8A2eMjy6H6RGsGADJQsnqkUgx9qi4qAZHhquGNUq
U6ZzxR9y57EXHGovBO34bl51mgAPaqCNw3rxf8OS9WrPxPQHENGXCp gBTOMHYwfGNWJMCa5pf6YI
yBbxwSgFGJjdFmdx3WukqVU4mH4HhtDsk2OPBWs8ZNvpmeXWprbUE6Q5XJVExcW JoC67CzSAVMI8mdBUgrbCEE1vs2yckuLYwTyTSHvxjT1RgdJTzrC3JT1xeQ
cAvqp5kbXGPMsIswkJeA4MRvyRKKj9pkv5HQ14Xajv6DqFuvX49HoGrdZ6r6I display
