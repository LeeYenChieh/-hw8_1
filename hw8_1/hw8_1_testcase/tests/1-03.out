x5XHuSL5LzOZ9TqiyojqF9qdz06vvP0Yt 
XgP 
